sale_force_invoiced_quantity/models/sale_order_line.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    force_invoiced = fields.Float(
```

Comment:
```suggestion
    force_invoiced_quantity = fields.Float(
```
is more explicit.

Reply:
Hi @rousseldenis, thank you for your feedback! Yup, sounds reasonable